src/System.Composition.AttributedModel/src/System.Composition.AttributedModel.csproj

```diff
@@ -6,7 +6,16 @@
     <RootNamespace>System.Composition</RootNamespace>
     <AssemblyName>System.Composition.AttributedModel</AssemblyName>
     <NuGetTargetMoniker Condition="'$(TargetGroup)' == ''">.NETStandard,Version=v1.0</NuGetTargetMoniker>
+    <AssemblyVersion>1.0.31.0</AssemblyVersion>
```

Comment:
I assume we picked this version because that was the version in the Microsoft.Composition package we had?